pwd_guess.py

```diff
@@ -288,7 +310,8 @@ def read_config_file(afile):
 class ModelDefaults(object):
     char_bag = (string.ascii_lowercase + string.ascii_uppercase +
                 string.digits + '~!@#$%^&*(),.<>/?\'"{}[]\\|-_=+;: `' +
-                PASSWORD_END)
+                PASSWORD_END + PASSWORD_START)
```

Comment:
I think that this would change the behavior of the filterer by allowing passwords that contain the tab character. Since you seem to be using the tab character as a special indicator for the model, I think this would mess things up with the new model. It would be better to somehow handle this internally to your new model code, by adding the tab character there.